TestProjects/UniversalGraphicsTest_2D/Packages/manifest.json

```diff
@@ -10,6 +10,7 @@
         "com.unity.render-pipelines.core": "file:../../../com.unity.render-pipelines.core",
         "com.unity.render-pipelines.universal": "file:../../../com.unity.render-pipelines.universal",
         "com.unity.shadergraph": "file:../../../com.unity.shadergraph",
+		"com.unity.testing.urp": "file:../../../com.unity.testing.urp",
```

Comment:
Nitpicky/minor formatting issue: there's an extra TAB (or at least on the github UI)

Reply:
(Same for l57)

Reply:
And all the .json files :)

Reply:
Arf, I edited these on notepad++, classic tabs vs spaces :( , fixing.